definitions/configuration.yaml

```diff
@@ -1,3 +1,12 @@
+data privacy requirements:
+  components:
+    - read date: dateperiod-1y
```

Comment:
@sumsumcity Please check this, I feel the best is to have a period attached to the date object